File: WEKEE.FE/UI.HOME/src/services/dto/searchOrderPageInput.ts
import { OrderByListInput } from "./orderByListInput";
export interface SearchOrderPageInput extends OrderByListInput {
    /// <summary>Tên trường tìm kiếm</summary>
    propertySearch: number[]
    /// <summary>Nội dung</summary>
    valuesSearch: String[]
}

File: WEKEE.BE/Account.Domain/ObjectValues/Input/SearchOrderPageInput.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace Account.Domain.ObjectValues.Input
{
    public class SearchOrderPageInput: OrderByPageListInput
    {
        /// <summary>Tên trường tìm kiếm</summary>
        public int[] PropertySearch { get; set; }
        /// <summary>Nội dung</summary>
        public string[] ValuesSearch { get; set; }
    }
}
